src/icon/icon.jsx

```diff
@@ -14,6 +14,7 @@ const Icon = ({
   title,
   nbPath,
   iconColor, /* set to true to print the native icon color */
+  alt, /* alternative text to vocalise the icon (only if no labelLeft or labelRight) */
```

Comment:
must update the comment, `alt` can be used with `labelLeft ` and `labelRight` to give more meaning to the accessible text